analyzer-alert/pom.xml

```diff
@@ -39,6 +39,12 @@
             <version>0.9.5.5</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.alibaba</groupId>
```

Comment:
There is already Jackson utils class, so fastjosn should not be relied on anymore